php-frontend/src/main/java/org/sonar/php/parser/NewPHPGrammar.java

```diff
@@ -43,13 +71,206 @@ public NewPHPGrammar(GrammarBuilder<InternalSyntaxToken> b, TreeFactory f) {
 
   public ExpressionTree EXPRESSION() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00100)